data_steward/tools/generate_combined_dataset.sh

```diff
@@ -152,8 +163,16 @@ export COMBINED_DATASET_ID="${combined_staging}"
 export BIGQUERY_DATASET_ID="${combined_staging}"
 data_stage='combined'
 
+# clear GOOGLE_APPLICATION_CREDENTIALS environment variable inorder to make impersonation work
+unset GOOGLE_APPLICATION_CREDENTIALS
+gcloud config set account "${pmi_email}"
+
 # run cleaning_rules on combined staging dataset
-python "${CLEANER_DIR}/clean_cdr.py" --project_id "${app_id}" --dataset_id "${combined_staging}" --sandbox_dataset_id "${combined_sandbox}" --data_stage ${data_stage} -s --cutoff_date "${ehr_cutoff}" --validation_dataset_id "${validation_dataset}" --ehr_dataset_id "${ehr_dataset}" --api_project_id "${api_project_id}" 2>&1 | tee combined_cleaning_log_"${combined}".txt
+python "${CLEANER_DIR}/clean_cdr.py" --project_id "${app_id}" --dataset_id "${combined_staging}" --sandbox_dataset_id "${combined_sandbox}" --run_as "${run_as}" --data_stage ${data_stage} -s --cutoff_date "${ehr_cutoff}" --validation_dataset_id "${validation_dataset}" --ehr_dataset_id "${ehr_dataset}" --api_project_id "${api_project_id}" 2>&1 | tee combined_cleaning_log_"${combined}".txt
+
+# Add GOOGLE_APPLICATION_CREDENTIALS environment variable
+export GOOGLE_APPLICATION_CREDENTIALS="${key_file}"
+gcloud auth activate-service-account --key-file=${key_file}
```

Comment:
Does this line need to be in the other ticket as well? #1189

Reply:
yes, I added it and di am gonna update that PR as well.